lib/clients/influxdb.js

```diff
@@ -1,19 +1,26 @@
 var dgram = require('dgram');
+var util = require('util');
+var BaseClient = require('./base');
 var _ = require('lodash');
-var async = require('async');
 
+/**
+ * A client that can send data to Influxdb backend via UDP
+ * @param {Object} opts options about the influxdb backend
+ * @param {String} opts.host the host of the influxdb. Default is 127.0.0.1
+ * @param {Number} opts.port the port of the UDP port of influxdb. Default is 4444.
+ * @param {Number} opts.sendQueueConcurrency specify the concurrency when send data to influxdb. Default is 10.
+ */
 var InfluxUdp = function influxUdp(opts) {
+  BaseClient.apply(this, arguments);
   opts = opts || {};
   var self = this;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the declaration of the unused 'self' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AUnusedVariable)